packages/connectivity_plus/connectivity_plus/README.md

```diff
@@ -55,10 +57,6 @@ if (connectivityResult.contains(ConnectivityResult.mobile)) {
 }
 ```
 
-> **Note**
->
-> You should not be using the current network status for deciding whether you can reliably make a network connection. Always guard your app code against timeouts and errors that might come from the network layer.
-
```

Comment:
I don't think that we need 2 notes, so merged them together and put on top of the file.